frontend/src/components/common/TooltipWrapper.tsx

```diff
@@ -40,7 +40,7 @@ const TooltipWrapper: React.FunctionComponent<ITooltipWrapperProps> = (props) =>
         </Tooltip>
       }
     >
-      {props.children ?? <></>}
+      <div>{props.children ?? <></>}</div>
```

Comment:
Ideally `props.children` would be an appropriate component type so that they don't need to be wrapped in a div.  But it probably would only cause possible styling issues.

Reply:
Ya I was trying to get around it and I don't like that I have to use the div but it doesn't seem to cause any problems as far as I can see.